pkg/cmd/kafka/acl/list/list.go

```diff
@@ -116,6 +121,9 @@ func NewListACLCommand(f *factory.Factory) *cobra.Command {
 	flags.AddUser(&userID)
 	flags.AddServiceAccount(&serviceAccount)
 	flags.AddAllAccounts(&allAccounts)
+	flags.AddCluster(&opts.cluster)
```

Comment:
I re-read the requirements and I could be wrong but I don't think it meant to show only cluster-scoped ACLs, rather be able to list ACLs per cluster:

```shell
rhoas kafka acl list --instance-id=xxxx
```

From the requirements:

> When I am using the `rhoas` CLI,

> I want to list the ACLs per Cluster, Topic and/or Group,

> So I can see which principals are allowed to perform which operations on a given Cluster, Topic and/or Group.

That to me means list the ACLs of a cluster.

Do you agree @rkpattnaik780 @wtrocki? If we are unsure we should clarify with PM.

Reply:
When you brought that it makes sense but initially I also got it as "cluster" type. 
My take is that more likely to be able to do it by cluster so we might remove it, but would wait for Duncan to confirm.

Reply:
Going through requirements I think the same, thanks for the catch!